pywps/configuration.py

```diff
@@ -17,6 +17,8 @@
 
 __author__ = "Calin Ciociu"
 
+from pywps.inout.basic import file_uri
```

Comment:
It appears that this is creating a circular import. You might need to find a different approach for loading this function (move to another file maybe)?

Reply:
done, thanks!
Where did you find the error message?

Reply:
You might still need to update some tests: 
```
 ImportError: cannot import name 'uri_to_path' from 'pywps.inout.basic' (/home/runner/work/pywps/pywps/pywps/inout/basic.py)
```

If you click on the checks below, you can see where each build is failing. Stack trace is there.

Reply:
thanks! somehow I've missed that stack trace :)

Reply:
I don't understand why only in 3.6 I get this error now...
`AttributeError: module 'pywps' has no attribute 'configuration'`